pkg/kubelet/cm/cpumanager/policy_static_test.go

```diff
@@ -660,7 +660,7 @@ func runStaticPolicyTestCase(t *testing.T, testCase staticPolicyTest) {
 				testCase.description, container.Name, st.assignments)
 		}
 
-		if !reflect.DeepEqual(cset, testCase.expCSet) {
+		if !cset.Equals(testCase.expCSet) {
 			t.Errorf("StaticPolicy Allocate() error (%v). expected cpuset %v but got %v",
```

Comment:
Maybe this out of the scope of this PR, but wouldn't it be to replace `%v` with `%s` for the `cpuset.CPUset` object and get nicer output?

Reply:
why not? adding